### PR TITLE
Long press on posts opening share options

### PR DESCRIPTION
**Pull request by brunodelgado**
This PR adds a long press gesture in the the posts list so it's easy to share as mentioned on #52 
Uses a category on UIViewController to prevent duplicating UIActivityViewController code (It's used in post details as well).

### CODE REVIEW

**Comment by leoformaggio** [comment]
@brunodelgado could you remove the spaces added for empty lines? Enabling the _Including whitespace-only lines_ option inside Text Editing tab on Xcode Preferences should work.

**Comment by brunodelgado** [comment]
Ops, my bad...
Done!

File: MacMagazine/Categories/UIViewController/UIViewController+ShareActivity.h [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#import <UIKit/UIKit.h>
```
Comment (fsaragoca) [inline MacMagazine/Categories/UIViewController/UIViewController+ShareActivity.h]: Usamos `@import UIKit;` no lugar de `#import`. Vamos tentar esforçar isso usando CI.

File: MacMagazine/Categories/UIViewController/UIViewController+ShareActivity.h [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+@interface UIViewController (ShareActivity)
+
+- (void)shareActivityItems:(NSArray *)activityItems
```
Comment (fsaragoca) [inline MacMagazine/Categories/UIViewController/UIViewController+ShareActivity.h]: Não sou muito fã de categories, eu preferia que fosse uma classe a parte dedicada pra isso, mas isso não é um blocker. Você pode prefixar os métodos com `mmm_`? Exemplo: `mmm_ shareActivityItems:completion:`.

File: MacMagazine/Categories/UIViewController/UIViewController+ShareActivity.h [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#import <UIKit/UIKit.h>
+
+@interface UIViewController (ShareActivity)
```
Comment (fsaragoca) [inline MacMagazine/Categories/UIViewController/UIViewController+ShareActivity.h]: Pode adicionar anotações de nullability?

File: MacMagazine/Classes/View Controllers/MMMPostsTableViewController.m [hunk 4]
```diff
@@ -206,6 +207,42 @@ - (CGFloat)tableView:(UITableView *)tableView heightForHeaderInSection:(NSIntege
     return [MMMTableViewHeaderView height];
 }
 
+#pragma mark - Long press gesture
+- (void)enableLongPressGesture
```
Comment (fsaragoca) [inline MacMagazine/Classes/View Controllers/MMMPostsTableViewController.m]: Adicionar uma linha em branco entre `#pragma` e `- (void)enableLongPressGesture`

**Comment by brunodelgado** [comment]
Sorry for taking so long to fix. I was a bit busy.
I've chosen the category because I saw a few of them and followed the practice, it's easy to change to specific class to handle the UIActivityViewController for the application.

What do you guys think?

**Pull request merged**